recipe/0001-Try-import-sklearn.impute.SimpleImputer-for-sklearn-0.22-compat.patch

```diff
@@ -0,0 +1,14 @@
+--- py_entitymatching/matcher/matcherutils.py
++++ py_entitymatching/matcher/matcherutils.py
+@@ -8,7 +8,10 @@
+ 
+ import pandas as pd
+ import sklearn.model_selection as ms
+-from  sklearn.preprocessing import Imputer
++try:
++    from sklearn.impute import SimpleImputer as Imputer
```

Comment:
This won't work since https://github.com/anhaidgroup/py_entitymatching/blob/v0.3.2/py_entitymatching/matcher/matcherutils.py#L221 uses the `axis` argument that is not supported by `SimpleImputer`.